src/components/CRow.vue

```diff
@@ -35,23 +64,52 @@ import api from '@/api';
 export default {
   name: 'CRow',
   props: {
+    priority: Number,
     message_id: String,
     name: String,
     args: Array,
-    kwargs: Object
+    kwargs: Object,
+    enqueued_datetime: String,
+    started_datetime: String,
+    end_datetime: String
   },
   data() {
     return {
       canCancel: true,
       onError: false,
       isCanceling: false,
       error: '',
+      nameState: this.name,
       txtBtnCancel: 'Cancel'
     };
   },
+  filters: {
+    elapsedTime(strTime) {
+      if (strTime == undefined) {
+        return '';
+      }
+      let intTime = Date.parse(strTime);
```

Comment:
You should use date objects not number of seconds. const date = new Date(strTime); then you can use the dateFns library to work with these dates https://date-fns.org/docs/Getting-Started.

Also you should parse the dates in the api return data, this wat you will not have to bother to parse dates everywhere in your app.

```
const getMessages = () => {
  return axios.get('/messages/states').then(res => res.data.result.map(parseMessage));
};

function parseMessage(rawMessage) {
  return {
    priority: rawMessage.priority,
    messageId: rawMessage.message_id,
    name: rawMessage.name,
    args: rawMessage.args,
    kwargs: rawMessage.kwargs,
    enqueuedDatetime: rawMessage.enqueued_datetime ? new Date(rawMessage.enqueued_datetime) : null,
    startedDatetime: rawMessage.started_datetime ? new Date(rawMessage.started_datetime) : null,
    endDatetime: rawMessage.end_datetime ? new Date(rawMessage.end_datetime) : null,
  };
}
```

I also chose to rename the fields to camel case as it's the norm in javascript.